### ISSUE TITLE
Missing types in structure panel after class defn

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#310]
plugin version: 098-2020,3

### Description
The structure panel omits the types `errpage`, `jobpage`, `breakpoint` and `session` for the following input.  Removing the 2 class definitions makes them reappear.

![image](https://user-images.githubusercontent.com/1253341/111731979-d39d2b80-8831-11eb-849c-2c503ac823c6.png)

```
class type ['a] page =
  object
    inherit GObj.widget
    method update : 'a -> unit
    method on_update : callback:('a -> unit) -> unit
    method data : 'a
  end

class type control =
  object
    method detach : unit -> unit
  end

type errpage = (int * string) list page
type jobpage = string CString.Map.t page
type breakpoint = {
  file : string;
}

type session = {
  buffer : GText.buffer;
}

let create_buffer () = ()
```

**Comment by giraud** [comment]
minimum code:
```ocaml
class type control =
  object
    method detach : unit -> unit
  end

type errpage = (int * string) list page
```

**Comment by giraud** [comment]
Fixed in 0.98.2

**Comment by jfehrle** [comment]
Looks good, thanks.
